_std/__build.dm

```diff
@@ -107,7 +107,8 @@ o+`        `-` ``..-:yooos-..----------..`
 //#define MAP_OVERRIDE_GEHENNA			// Warcrimes WIP do not use
 //#define MAP_OVERRIDE_PAMGOC			// Pamgoc
 //#define MAP_OVERRIDE_WRESTLEMAP   // Wrestlemap by Overtone
-//#define MAP_OVERRIDE_POD_WARS   // 500x500 Pod Wars map
+// #define MAP_OVERRIDE_POD_WARS   // 500x500 Pod Wars map
+//
```

Comment:
Clear out the two backslashes on an empty line and you're golden.

Reply:
Done